validations/kyc.test.ts

```diff
@@ -58,6 +64,52 @@ describe('/kyc', () => {
       Object.values(KycStatus),
     )
   })
+  if (config.clientApiKey && config.providerId) {
```

Comment:
this sort of leaves the impression that webhooks are optional, but they're required by the spec. Put another way, I could be "passing all tests" without having ever thought about webhooks.

I see why you'd want to allow someone to run the tests without defining these (if they're implementing webhooks last or something), but they should know they're not "passing all tests" if that's what they do. Maybe this test could just fail right away if the configuration parameters aren't defined.